jekyll/_cci1/ios-builds-on-os-x.md

```diff
@@ -317,7 +317,7 @@ your organization's settings page.
 
 ![](  {{ site.baseurl }}/assets/img/docs/fabric-org-settings-page.png)
 
-Click on your organisation (CircleCI in the image above), and click on
+Click on your organization (CircleCI in the image above), and click on
```

Comment:
As mentioned in the extended comment of my commit, I said:

>Fix localization mistake. Documentation uses US English spelling and in this document we were using 'organisation' not 'organization'. Organisation is primarily only used in UK English.

To quote http://writingexplained.org/organisation-vs-organization-difference, Organisation is the correct spelling in British English and organization is the correct spelling in American English. However, the opposite isn't true: Organisation isn't the correct spelling in American English and Organization isn't the correct spelling in British English.